doctr/utils/geometry.py

```diff
@@ -44,3 +45,12 @@ def resolve_enclosing_bbox(bboxes: List[BoundingBox]) -> BoundingBox:
 def resolve_enclosing_rbbox(rbboxes: List[RotatedBbox]) -> RotatedBbox:
     pts = np.asarray([pt for rbbox in rbboxes for pt in rbbox_to_polygon(rbbox)], np.float32)
     return fit_rbbox(pts)
+
+
+def resolve_enclosing_bboxarray(bboxarray: np.ndarray) -> np.ndarray:
+    """Compute enclosing bbox from an array of boxes: (*, 5), where boxes have this shape:
+    (xmin, ymin, xmax, ymax, score).
+    Return a (1, 5) array (enclosing boxarray)
+    """
+    xmin, ymin, xmax, ymax, score = np.split(bboxarray, 5, axis=1)
+    return np.array([xmin.min(), ymin.min(), xmax.max(), ymax.max(), score.mean()])
```

Comment:
Actually, since both functions are doing the same, could you update the original function and only add support for numpy arrays?

something like that for the signature 
```python
def resolve_enclosing_bbox(bboxes: Union[List[BoundingBox], np.ndarray]) -> Union[BoundingBox, np.ndarray]:
```

That would make things simpler I guess

Reply:
using `if isinstance(bboxes, np.ndarray):` to check the type

Reply:
yes